frontend/docs/user-guide/administration-pages.md

```diff
@@ -17,13 +17,12 @@ associated with the suspended user.
 
 ## Configuration
 
-Administrators can view the current configurations for {{ $params.APPLICATION_NAME }}. If configuration settings need to be changed,
-a system admin developer will need to make those modifications.
+See the [configuration page documentation](./configuration.md).
```

Comment:
I would provide at least a little bit of detail here and then a "For more information, see the configuration page documentation."

They should be able to still get a high-level overview from this page I think.